cacher/cacher.go

```diff
@@ -134,6 +136,18 @@ func (cacher *Cacher) dispatch(id [32]byte, msg http.RequestWithResponder) {
 				if tx.Tx.Output.String() == pack.NewTyped().String() {
 					return true
 				}
+
+				var output engine.LockMintBurnReleaseOutput
+				err = pack.Decode(&output, tx.Tx.Output)
+				if err != nil {
+					// it might a different kind of output, so it's ok
```

Comment:
Might be better to only do this if the transaction has a cross-chain selector (`tx.Tx.Selector.IsCrossChain()`). That way we can handle the error properly.